samples/sensor/dht_polling/README.rst

```diff
@@ -0,0 +1,49 @@
+.. _dht_polling:
+
+Generic Digital Humidity Temperature sensor polling sample
+##########################################################
+
+Overview
+********
+
+This sample application demonstrates how to use digital humidity temperature
+sensors.
+
+Building and Running
+********************
+
+This sample supports up to 10 humidity/temperature sensors. Each sensor needs to
+be aliased as ``dhtN`` where ``N`` goes from ``0`` to ``9``. For example:
+
+.. code-block:: devicetree
+
+  / {
+	aliases {
+			dht0 = &hs300x;
+		};
+	};
+
+Make sure the aliases are in devicetree, then build and run with:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/sensor/dht_polling
+   :board: <board to use>
+   :goals: build flash
+   :compact:
+
+Sample Output
+=============
+
+.. code-block:: console
+
+       hs300x@44: temp is 25.31129 oC humidity is 30.391259 %RH
+       hs300x@44: temp is 25.51272 oC humidity is 30.446194 %RH
+       hs300x@44: temp is 25.51272 oC humidity is 30.372947 %RH
+       hs300x@44: temp is 25.51272 oC humidity is 30.391259 %RH
+       hs300x@44: temp is 25.31129 oC humidity is 30.372947 %RH
+       hs300x@44: temp is 25.31129 oC humidity is 30.354635 %RH
+       hs300x@44: temp is 25.51272 oC humidity is 30.372947 %RH
+       hs300x@44: temp is 25.51272 oC humidity is 30.372947 %RH
+       hs300x@44: temp is 25.51272 oC humidity is 30.391259 %RH
+       hs300x@44: temp is 25.51272 oC humidity is 30.446194 %RH
+       hs300x@44: temp is 25.51272 oC humidity is 30.531648 %RH
```

Comment:
update to use "°" and possibly the newer format as per my other comment.

Reply:
done, but see comment above.